pokeapp/src/components/FilterDropDown.js

```diff
@@ -142,7 +143,10 @@ const FilterDropDown = ({
     return (
       <div
         name={sortTypeEnum}
-        onClick={() => addRemoveTypeFilter(sortTypeEnum)}
+        onClick={(event) => {
+          event.stopPropagation();
```

Comment:
interesting...

Reply:
Since this clickable container sits on top of another clickable container, we only want this onClick to fire, not the parent's, so we're using `stopPropagation()` to prevent the parent from responding to it.